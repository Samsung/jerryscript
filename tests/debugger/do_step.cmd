break do_step.js:25
step
backtrace
b do_step.js:26
next
next
bt
c
