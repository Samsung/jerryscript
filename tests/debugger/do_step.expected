Connecting to: localhost:5001
Stopped at tests/debugger/do_step.js:32:1
(jerry-debugger) step
Stopped at tests/debugger/do_step.js:22:7 (in f1() at line:15, col:1)
(jerry-debugger) step
Stopped at tests/debugger/do_step.js:19:5 (in g() at line:17, col:3)
(jerry-debugger) backtrace
Frame 0: tests/debugger/do_step.js:19:5 (in g() at line:17, col:3)
Frame 1: tests/debugger/do_step.js:22:12 (in f1() at line:15, col:1)
Frame 2: tests/debugger/do_step.js:32:3
(jerry-debugger) next
Stopped at tests/debugger/do_step.js:23:3 (in f1() at line:15, col:1)
(jerry-debugger) bt
Frame 0: tests/debugger/do_step.js:23:3 (in f1() at line:15, col:1)
Frame 1: tests/debugger/do_step.js:32:3
(jerry-debugger) next
Stopped at tests/debugger/do_step.js:33:1
(jerry-debugger) s
Stopped at tests/debugger/do_step.js:28:3 (in f2() at line:26, col:1)
(jerry-debugger) n
Stopped at tests/debugger/do_step.js:29:3 (in f2() at line:26, col:1)
(jerry-debugger) bt
Frame 0: tests/debugger/do_step.js:29:3 (in f2() at line:26, col:1)
Frame 1: tests/debugger/do_step.js:33:3
(jerry-debugger) c
